src/blueDarkTheme.ts

```diff
@@ -6,88 +6,83 @@ All rights reserved.
 This code is licensed under the BSD-3 license found in the LICENSE file in the root directory of this source tree and at https://opensource.org/licenses/BSD-3-Clause.
 **/
 
-import { configureFonts, DefaultTheme } from 'react-native-paper';
-import { blue, red, darkBlack, black, white, lightBlue } from '@brightlayer-ui/colors';
-import { fontConfig, ThemeOpacity } from './shared';
+import { MD3DarkTheme, configureFonts } from 'react-native-paper';
+import { fontConfig } from './shared';
+import { BLUIColors } from '@brightlayer-ui/colors';
 import Color from 'color';
 
-const themeOpacity: ThemeOpacity = {
-    disabled: 0.36,
-    divider: 0.36,
-    disabledBackground: 0.24,
-    actionOpacity: 0.1,
-};
-
-export const blueDarkTheme: ReactNativePaper.Theme = {
-    ...DefaultTheme,
-    dark: true,
-    roundness: 4,
-    fonts: {
-        ...configureFonts(fontConfig),
-        bold: {
-            fontFamily: 'OpenSans-Bold',
-            fontWeight: '700',
-        },
-    },
+export const MD3BluiDarkTheme = {
+    ...MD3DarkTheme,
+    fonts: configureFonts({ config: fontConfig }),
     colors: {
```

Comment:
This is a great big black hole of color definitions without context for any of the values. I recommend separating the keys that are part of RNP (move them to the top of the list and and a comment that marks them as such) and then have a separate section (with a comment to identify them) for the color definitions that we have added. And each color that we have added needs a comment explaining why it was added and where it is intended to be used.

Reply:
Can you please let me know, where can I find the information on each custom colors, why and where it is intended to be used?

Reply:
@joebochill that's a design thing. I'd need to put together an instruction on docit for designers for how to use colors properly too. We can skip this black hold color definition comment thing for now.

Reply:
Wow, Manoj you are quick! :D

Reply:
I'll add a task to add the comments for next sprint. If we are defining a custom color in the theme, presumably we are using it somewhere...or intend to use it somewhere...otherwise we wouldn't be adding them.